apps/src/imageUtils.js

```diff
@@ -177,3 +177,13 @@ export async function toImageData(input) {
     throw new Error('Unable to convert input to ImageData: ' + err);
   }
 }
+
+/**
+ * @param {Blob}
+ */
+export function downloadBlobAsPng(blob) {
+  const download = document.createElement('a');
+  download.href = URL.createObjectURL(blob);
+  download.download = 'image.png';
```

Comment:
the filename should probably be configurable -- for now, it could be a second arg to this method that has a default value